src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -141,6 +141,19 @@ private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appCo
                     }
                 }
             }
+            else
+            {
+                string errorMessage = $"The feature declaration for the feature '{feature}' was not found.";
+
+                if (!_options.IgnoreMissingFeatures)
+                {
+                    throw new FeatureManagementException(FeatureManagementError.MissingFeatureFilter, errorMessage);
```

Comment:
Should this error type be `FeatureManagementError.MissingFeature`?

Reply:
Yes, thanks. Here's a PR to fix: https://github.com/microsoft/FeatureManagement-Dotnet/pull/142